app/src/pages/header/components/AddMovieButton.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Button = styled.button`
+    border: none;
+    border-radius: 4px;
+    color: #f65261;
+    background: rgba(255, 255, 255, 0.2);     
+    padding: 10px 15px;
+`
+
+function AddMovieButton(){
```

Comment:
Do we need to add space after function call ():

```
function AddMovieButton() {
    someAwesomeCode();
}
```

Reply:
Ooh, interesting question) let's ask it